homeassistant/components/stream/worker.py

```diff
@@ -300,7 +316,11 @@ def peek_first_dts() -> bool:
                                 _LOGGER.warning(
                                     "ADTS AAC detected - disabling audio stream"
                                 )
-                                container_packets = container.demux(video_stream)
+                                dts_validator = TimestampValidator()
```

Comment:
Can we just reuse the existing dts_validator? This would also prevent an out of order packet occurring between `initial_packets` and `container_packets`

Reply:
Done.